src/main/groovy/life/qbic/portal/sampletracking/app/samples/list/ModifySampleList.groovy

```diff
@@ -0,0 +1,48 @@
+package life.qbic.portal.sampletracking.app.samples.list
+
+import groovy.util.logging.Log4j2
+import life.qbic.datamodel.samples.Sample
+
+@Log4j2
+class ModifySampleList implements ModifySampleListInput {
+
+    ModifySampleListOutput output
+
+    ModifySampleList() {
+
```

Comment:
This constructor is dangerous in the sense, that it can be called, and the field `output` remains null (which can then later cause ugly NPEs and thus should be avoided).

Reply:
so I suggest to either implement a `DefaultModifySampleList` that will prevent NPEs if no output is specified or make this constructor private.

Reply:
Yes initialisation with an empty List should be done in the default constructor.